components/dashboard/src/App.tsx

```diff
@@ -4,188 +4,191 @@
+             <a className="mx-auto" href="mailto:support@gitpod.io?Subject=Blocked"><button className="secondary">Contact Support</button></a>
+         </div>;
+     }
+     const shouldWhatsNewShown = shouldSeeWhatsNew(user)
+     if (shouldWhatsNewShown !== isWhatsNewShown) {
+         setWhatsNewShown(shouldWhatsNewShown);
+     }
+     
+     window.addEventListener("hashchange", () => {
+         // Refresh on hash change if the path is '/' (new context URL)
+         if (window.location.pathname === '/') {
+             window.location.reload(true);
+         }
+     }, false);
+ 
+     let toRender: React.ReactElement = <Route>
+         <div className="container">
+             {renderMenu(user)}
+             <Switch>
+                 <Route path="/workspaces" exact component={Workspaces} />
+                 <Route path="/account" exact component={Account} />
+                 <Route path="/integrations" exact component={Integrations} />
+                 <Route path="/notifications" exact component={Notifications} />
+                 <Route path="/plans" exact component={Plans} />
+                 <Route path="/teams" exact component={Teams} />
+                 <Route path="/variables" exact component={EnvironmentVariables} />
+                 <Route path="/preferences" exact component={Preferences} />
+                 <Route path="/install-github-app" exact component={InstallGitHubApp} />
+                 <Route path="/from-referrer" exact component={FromReferrer} />
+                 
+                 <Route path="/admin/users" component={UserSearch} />
+                 <Route path="/admin/workspaces" component={WorkspacesSearch} />
+ 
+                 <Route path={["/", "/login"]} exact>
+                     <Redirect to="/workspaces"/>
+                 </Route>
+                 <Route path={["/settings"]} exact>
+                     <Redirect to="/account"/>
+                 </Route>
+                 <Route path={["/access-control"]} exact>
+                     <Redirect to="/integrations"/>
+                 </Route>
+                 <Route path={["/subscription", "/usage", "/upgrade-subscription"]} exact>
+                     <Redirect to="/plans"/>
+                 </Route>
+                 <Route path={["/admin"]} exact>
+                     <Redirect to="/admin/users"/>
+                 </Route>
+                 <Route path="/sorry" exact>
+                     <div className="mt-48 text-center">
+                         <h1 className="text-gray-500 text-3xl">Oh, no! Something went wrong!</h1>
+                         <p className="mt-4 text-lg text-gitpod-red">{decodeURIComponent(getURLHash())}</p>
+                     </div>
+                 </Route>
+                 <Route path="*" /* status={404} */>
```

Comment:
<details>
<summary>:warning: <strong>Commented-out Code is Committed.</strong> (click for details)</summary>

This pull request contains commented code which is not recommended when a version control repository (e.g., GitHub) is used. It is recommended to delete all commented instructions before committing to VCS.

**[More details](https://confluence.devfactory.com/x/yAV7HQ)**

## The improvement suggestion in this case
Consider avoiding committing commented code.
</details>

[//]: # (<a id="c483fce2-a802-4337-b694-0393c5c08620" data-finding-id="494601672" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)